### DISCUSSION TITLE
Filament on production stuck on 302 Http redirection loop

### DISCUSSION THREAD

**Discussion by AzizRaad** [filamentphp/filament#12893] [Help]
### Package

Panel builder

### Package Version

v3.2

### How can we help you?

I'm using latest Filament php , and I created one panel with the name /superadmin , it works perfectly on local, once I deployed it do production I suddenly can't view the page anymore and whenever I try to access the page it redirects me to the main page " / " 
notes:
- I ran composer update so all packages are installed and in place
- I implemented the "canAccessPanel" method in my Model (its not an auth issue else it would return 403 forbidden)

**Comment by leandrocfe** [comment]
Any filament customization? Please provide more info..

**Reply by AzizRaad** [reply]
no customization, no plugins, I just installed filament.
The latest update I got to is: 
it might be something related to " canAccesPanel " when I implemented it and return "true" it works just fine
but if I try to return any kind of other if statements it also doesn't work
